biojava-structure-gui/src/main/java/org/biojava/nbio/structure/align/gui/autosuggest/SCOPAutoSuggestProvider.java

```diff
@@ -179,7 +183,7 @@ public void clear() {
 	public void stop() {
 		stop.set(true);
 		if (DEBUG)
```

Comment:
Here we can either get rid of the DEBUG variable altogether or use LOGGER.isDebugEnabled(). The best solution is not having DEBUG at all and using the LOGGER.debug statements directly.